style.css

```diff
@@ -304,3 +306,79 @@ li {
   color: #fff;
   padding: 0 0 0 40px;
 }
+
+.contactme {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 30px;
+  margin: 1vh;
+}
+
+form {
+  border: none;
+  background-color: transparent;
+  resize: none;
+  outline: none;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+}
+
+.contact--input {
+  border-style: hidden;
+  border-bottom: solid 1px #ccc;
+  background-color: rgb(255, 255, 255);
+  padding-top: 25px;
+  margin-top: 30px;
+  text-indent: 20px;
+  padding-bottom: 20px;
+}
```

Comment:
- [ ] Please hide the border that appears when the text field is active. You can use `:focus` on the selector and add `border: none` inside the block.